[2023-08-22T03:02:51.397+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Investimento_api.Extracao_Invest manual__2023-08-22T03:02:47.640251+00:00 [queued]>
[2023-08-22T03:02:51.404+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Investimento_api.Extracao_Invest manual__2023-08-22T03:02:47.640251+00:00 [queued]>
[2023-08-22T03:02:51.405+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-08-22T03:02:51.416+0000] {taskinstance.py:1382} INFO - Executing <Task(SparkSubmitOperator): Extracao_Invest> on 2023-08-22 03:02:47.640251+00:00
[2023-08-22T03:02:51.421+0000] {standard_task_runner.py:57} INFO - Started process 113 to run task
[2023-08-22T03:02:51.425+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Investimento_api', 'Extracao_Invest', 'manual__2023-08-22T03:02:47.640251+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpfia66eh9']
[2023-08-22T03:02:51.428+0000] {standard_task_runner.py:85} INFO - Job 174: Subtask Extracao_Invest
[2023-08-22T03:02:51.471+0000] {task_command.py:415} INFO - Running <TaskInstance: Investimento_api.Extracao_Invest manual__2023-08-22T03:02:47.640251+00:00 [running]> on host 378eff3ef0da
[2023-08-22T03:02:51.542+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Investimento_api' AIRFLOW_CTX_TASK_ID='Extracao_Invest' AIRFLOW_CTX_EXECUTION_DATE='2023-08-22T03:02:47.640251+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-22T03:02:47.640251+00:00'
[2023-08-22T03:02:51.551+0000] {base.py:73} INFO - Using connection ID 'spark_standalone' for task execution.
[2023-08-22T03:02:51.553+0000] {spark_submit.py:340} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --name Extracao_Invest ../operators/Raw_Operator.py
[2023-08-22T03:02:53.028+0000] {spark_submit.py:491} INFO - WARNING: An illegal reflective access operation has occurred
[2023-08-22T03:02:53.029+0000] {spark_submit.py:491} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.8/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2023-08-22T03:02:53.030+0000] {spark_submit.py:491} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2023-08-22T03:02:53.030+0000] {spark_submit.py:491} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2023-08-22T03:02:53.031+0000] {spark_submit.py:491} INFO - WARNING: All illegal access operations will be denied in a future release
[2023-08-22T03:02:53.438+0000] {spark_submit.py:491} INFO - python3: can't open file '/opt/operators/Raw_Operator.py': [Errno 2] No such file or directory
[2023-08-22T03:02:53.453+0000] {spark_submit.py:491} INFO - log4j:WARN No appenders could be found for logger (org.apache.spark.util.ShutdownHookManager).
[2023-08-22T03:02:53.454+0000] {spark_submit.py:491} INFO - log4j:WARN Please initialize the log4j system properly.
[2023-08-22T03:02:53.454+0000] {spark_submit.py:491} INFO - log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
[2023-08-22T03:02:53.486+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 156, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 422, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --name Extracao_Invest ../operators/Raw_Operator.py. Error code is: 2.
[2023-08-22T03:02:53.490+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=Investimento_api, task_id=Extracao_Invest, execution_date=20230822T030247, start_date=20230822T030251, end_date=20230822T030253
[2023-08-22T03:02:53.505+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 174 for task Extracao_Invest (Cannot execute: spark-submit --master spark://spark:7077 --name Extracao_Invest ../operators/Raw_Operator.py. Error code is: 2.; 113)
[2023-08-22T03:02:53.524+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-22T03:02:53.549+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
